src/eip7594/fk20.c

```diff
@@ -25,14 +25,30 @@
 /**
  * Reorder and extend polynomial coefficients for the toeplitz method, strided version.
  *
- * @param[out]  out     The reordered polynomial, length `CELLS_PER_EXT_BLOB`
+ * This is an auxiliary function that selects the polynomial coefficient for the circulant matrix
+ * in the FK20 multiproof algorithm (Section 3)
+ *
+ * The constants in this function correspond to the FK20 notation as follows:
+ * FIELD_ELEMENTS_PER_CELL =  `l`
+ * CELLS_PER_BLOB = `r`
+ * CELLS_PER_EXT_BLOB = `n`
+ * FIELD_ELEMENTS_PER_BLOB = `d` +1
+ * @offset = `i`
+ *
+ * This function outputs the first column of the circulant matrix `F''_i`,
+ * The matrix `F''_i` is the padding of the Toeplitz matrix of size (r-1)*(r-1) to
+ * the size 2r*2r
+ *
+ * It is supposed to output (in[d-i],0,...,0 (`r` zeros),p[d-(r-2)l-i,p[d-(r-3)l-i,...,p[d-l-i])
+ *
+ * @param[out]  out     The reordered polynomial, length `2*CELLS_PER_BLOB`
  * @param[in]   in      The input polynomial, length `FIELD_ELEMENTS_PER_BLOB`
- * @param[in]   offset  The offset
+ * @param[in]   offset  The offset, the integer between 0 and FIELD_ELEMENTS_PER_BLOB-1, inclusive
```

Comment:
Can we document what this "offset" does in practice?

Reply:
See lines 35 and 41